cinema/models.py

```diff
@@ -134,9 +137,8 @@ def save(
         )
```

Comment:
In your custom `save` method, you are calling `full_clean` which is a good practice as it includes calling `clean`. However, keep in mind that this can lead to performance issues if the save method is called very frequently, as validation can be a costly operation.